### DISCUSSION TITLE
Changes required if SSL certificate need to be changed

### DISCUSSION THREAD

**Discussion by vayu068** [Sunbird-Ed/Community#229] [Installation] [answered]
Hi, Can you please share what services need to be redeployed if there is a need to change SSL certificate in a running setup?

**Comment by beepdot** [comment, marked as answer]
Hello @vayu068 
Please update the SSL in your private repo and just run `BootstrapMinimal` jenkins jobs. After this restart the `nginx-public-ingress` daemonset.

**Reply by vayu068** [reply]
Thanks, will try this.
